src/Main.java

```diff
@@ -1,5 +1,61 @@
+import java.time.LocalDate;
+
 public class Main {
+    public static void checkingLeapYear(int year2) {
+        if (year2 % 4 != 0) {
+            System.out.println(year2 + " год не является високосным.");
+        } else if (year2 % 100 == 0 && year2 % 400 != 0) {
+            System.out.println(year2 + " год не является високосным.");
+        } else {
+            System.out.println(year2 + " год является високосным.");
+        }
+        System.out.println();
+    }
+
+    public static void ResolvesInstallingOS(int clientOS2, int clientDeviceYear2) {
+
+        if (clientOS2 == 0) {
+            if (clientDeviceYear2 < 2015) {
+                System.out.println("Установите облегченную версию приложения для iOS по ссылке--->");
+            } else {
+                System.out.println("Установите версию приложения для iOS по ссылке --->");
+            }
+        } else {
+            if (clientDeviceYear2 < 2015) {
+                System.out.println("Установите облегченную версию приложения для Android по ссылке--->");
+            } else {
+                System.out.println("Установите версию приложения для Android по ссылке --->");
+            }
+        }
+        System.out.println();
+    }
+
+    public static int calculationDeliveryDistance(int deliveryDistance2) {
+        int DeliveryTime;
```

Comment:
Имена переменных строго с маленькой буквы в camelCase
```suggestion
        int deliveryTime;
```